CRM/Custom/Page/Option.php

```diff
@@ -167,8 +167,8 @@ public function browse() {
         [1 => $reusedNames]
       );
       CRM_Utils_System::setTitle($newTitle);
-      $this->assign('reusedNames', $reusedNames);
     }
+    $this->assign('reusedNames', !empty($reusedNames) ? $reusedNames :  NULL);
```

Comment:
@eileenmcnaughton there's a style warning for the extra space before `NULL`. But also, wouldn't this be simpler?
```suggestion
    $this->assign('reusedNames', $reusedNames ?? NULL);
```

Reply:
@colemanw so `$reusedNames` would be an empty array at this point but I think it's kinda supposed tobe assigned as NULL to the template

Reply:
Maybe it should be string `'null'`...

Seriously it looks like it should be a string in the tpl but null works for the `{if}` block.